git-branchless/src/revset/grammar.lalrpop

```diff
@@ -106,5 +106,5 @@ Symbol2: String = {
 }
 
 Symbol1: String = {
-    r"[a-zA-Z0-9/_$@.]+" => <>.to_string(),
+    r"[a-zA-Z0-9/_$@.-]+" => <>.to_string(),
```

Comment:
In what contexts will `-` be interpreted as an operator (for set difference) after this change?

Reply:
It looks like it's only interpreted as an operator when there is a space to disambiguate, which is what I was hoping for.